### DISCUSSION TITLE
[@types/docusign-esign] `EnvelopesApi#getDocument` incorrect return type.

### DISCUSSION THREAD

**Discussion by xdc0** [DefinitelyTyped/DefinitelyTyped#57452] [Issues with a @types package]
`EnvelopesApi#getDocument` has an incorrect return type. It is currently returning `Promise<object>` but in practice it returns `Promise<string>`.

[a comment by an automated account is not shown]

**Comment by equimbo** [comment]
Fixed the return type for getEnvelope method.   [Pull request](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57459) in progress...

**Comment by equimbo** [comment]
This is now fixed
